tox.ini

```diff
@@ -102,6 +94,19 @@ commands_pre = rm -rf dist/
 changedir = tests/non-pytest/poetry-lock-test
 commands = poetry lock
 
+[testenv:freezedeps]
```

Comment:
Guess what! You can modify the header output by modifying this environment variable:

### `CUSTOM_COMPILE_COMMAND`

https://github.com/jazzband/pip-tools/blob/6735e6271f489a73f490a1b28dd47f2616417e73/piptools/writer.py#L121-L134

This will allow you to put the tox command directly into these files. I think you can then remove pip-compile-multi and just use pip-tools exclusively.

Reply:
I found that this isn't supported by pip-compile-multi, the pip-compile-multi replacement option is for setting the whole header, and overall I'm not loving my experience trying to get this set for both tools.
I'm going to reduce this down to `pip-compile`-only and set the `CUSTOM_COMPILE_COMMAND`.